exchange/exchange-ps/exchange/Set-TransportRule.md

```diff
@@ -1315,6 +1315,10 @@ Accept wildcard characters: False
 ```
 
 ### -DlpPolicy
+
+Existing Transport rules with this property are no longer being evaluated or executed because it is linked to an Exchange DLP policy. 
+Please migrate the policies to the M365 Compliance Center by using the migration wizard.
```

Comment:
```suggestion
Existing transport rules with this property are no longer evaluated or executed because it is linked to an Exchange DLP policy. 
You should migrate the policies to the Microsoft 365 compliance center by using the migration step-by-step guide.
```